docs/standard/conventions.md

```diff
@@ -0,0 +1,87 @@
+# Schema conventions
+
+This page documents conventions that MUST or SHOULD be applied when authoring OCDS schema or extensions.
+
+## Definitions section
+
+The top level of the schema is split between `properties` (the basic structure of an OCDS release) and `definitions` (containing objects that may be re-used in multiple locations across the schema) which are referenced whenever they are used. 
+
+Each of the objects in `definitions` can be thought of as a 'Class', and it's name is capitalised accordingly. 
+ 
+Whenever you consider that an object or structure might be re-used in a different area of the standard, it should be included in `definitions`, and referenced. 
+
+## Naming conventions
+
+We use lower [camelCase](https://en.wikipedia.org/wiki/Camel_case) for property names. E.g. `awardCriteriaDetails`.
+
+We use upper [CamelCase](https://en.wikipedia.org/wiki/Camel_case) for objects directly nested within in the `definitions` section. E.g. `Award`.
+
+We put the qualifier *before* the concept. E.g. `enquiryPeriod` rather than `periodOfEnquiry`.
+
+We use singular for properties pointing to an object or literal value.
+
+We use plural for properties pointing to an array of values. 
```

Comment:
May want to pull this section of the style guide into here: http://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/style_guide/#schema-style-guide

A bullet list may be better than a series of short paragraphs.